### PR TITLE
MGMT-11259: assisted-installer: Add Assisted Installer fast forwards for ocm-2.7 release

### PR DESCRIPTION
**Pull request by eliorerz**
Preparation for the ocm 2.7 FF

### CODE REVIEW

**Review by osherdp — CHANGES_REQUESTED** [review]
We should do the same for capi, no?

**Review by osherdp — APPROVED** [review]

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -2026,8 +2026,53 @@ periodics:
...
+        secretName: github-credentials-openshift-merge-robot
+- agent: kubernetes
+  cluster: app.ci
+  cron: '@hourly'
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  name: periodic-openshift-release-fast-forward-cluster-api-provider-agent
```
Comment (eliorerz) [inline ci-operator/jobs/infra-periodics.yaml:2059]: FYI @eranco74

**Comment by eliorerz** [comment]
/unhold
https://github.com/openshift/release/pull/30788 is merged

**Pull request merged**